src/app/workflows/workflows.module.ts

```diff
@@ -46,7 +47,8 @@ import { workflowsRouting } from './workflows.routing';
     TabsModule.forRoot(),
     TooltipModule.forRoot(),
     WorkflowModule,
-    workflowsRouting
+    workflowsRouting,
+    NgxJsonLdModule
```

Comment:
Remove the NgxJsonLdModule from these 4 modules and throw it into EntryModule (and export inside it) and see what happens.